spec/queries/mentoring_query_spec.rb

```diff
@@ -6,25 +6,74 @@
   describe '#call' do
     subject(:call) { described_class.new.call }
 
-    context 'when there are mentees and mentors' do
+    context 'when there are one mentee and mentor' do
       let(:mentor) { create(:user) }
       let!(:mentee) { create(:user, reviewer_id: mentor.id) }
 
-      let(:mentors) { call.map(&:mentors) }
+      let(:mentors) { call.map(&:name) }
       let(:offices) { call.map(&:city) }
       let(:mentees) { call.map(&:mentees) }
```

Comment:
although DRY is not always the way to go when it comes to testing, you could have these 3 calls outside of the `contexts` and leave it under the `describe` scope, so all the contexts can have access to it.

Reply:
Nice!